glab ci status failing (glab version 1.17.0)

**Description**
Getting error when running `status` subcommand on `glab ci`. Meanwhile `glab ci view` working as expected.
**Expected Behavior vs Actual Behavior**
Show status of latest pipeline with option to view logs & retry.

**Logs**
```bash
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xc55aaa]

goroutine 1 [running]:
github.com/profclems/glab/commands/ci/status.NewCmdStatus.func1(0xc000666280, 0x16bdda8, 0x0, 0x0, 0x0, 0x0)
        /home/runner/work/glab/glab/commands/ci/status/status.go:88 +0x49a
github.com/spf13/cobra.(*Command).execute(0xc000666280, 0x16bdda8, 0x0, 0x0, 0xc000666280, 0x16bdda8)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc0004e0000, 0xc0000a8008, 0xd3, 0x2)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x349
main.main()
        /home/runner/work/glab/glab/cmd/glab/main.go:140 +0x793
```
**Your Environment**
Ubuntu 20.04 running as a Docker container on Amazon Linux

* Version used (Run `glab --version`): 1.17.0
* Operating System and version: Ubuntu 20.04

--- a/commands/ci/status/status.go
+++ b/commands/ci/status/status.go
@@ -85,7 +85,12 @@ func NewCmdStatus(f *cmdutils.Factory) *cobra.Command {
 						if job.FinishedAt != nil {
 							end = *job.FinishedAt
 						}
-						duration := utils.FmtDuration(end.Sub(*job.StartedAt))
+						var duration string
+						if job.StartedAt != nil {
+							duration = utils.FmtDuration(end.Sub(*job.StartedAt))
+						} else {
+							duration = "not started"
+						}
 						var status string
 						switch s := job.Status; s {
 						case "failed":

